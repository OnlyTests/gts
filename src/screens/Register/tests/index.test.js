import React from 'react';
import {render} from '@testing-library/react-native';
import RegisterScreen from '../index';
describe('RegisterScreen, check if render success', () => {
  test('render', async () => {
    const {toJSON, unmount} = await render(<RegisterScreen />);
    expect(toJSON()).toMatchSnapshot();
    await unmount();
  });
});
