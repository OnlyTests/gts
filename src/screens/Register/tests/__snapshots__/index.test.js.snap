// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterScreen, check if render success render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "#F5F6F7",
      },
    ]
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#F8F8F8",
              "borderBottomColor": "#a7a6ab",
              "borderBottomWidth": 0.5,
              "elevation": 3,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "right": 0,
              "shadowColor": undefined,
              "shadowOffset": undefined,
              "shadowOpacity": undefined,
              "shadowRadius": undefined,
              "top": 0,
            },
            Object {
              "backgroundColor": undefined,
              "elevation": 0,
              "height": 45,
              "justifyContent": "flex-start",
              "paddingTop": 0,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "flex-start",
              "alignSelf": "center",
              "flex": 1,
            }
          }
        >
          <TouchableOpacity
            onPress={[Function]}
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#8f8f8f",
                    },
                  ],
                  Object {
                    "fontFamily": "Ionicons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
            >
              ï„¤
            </Text>
          </TouchableOpacity>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "flex": 1,
              },
              Object {
                "flex": 3,
              },
            ]
          }
        >
          <Text
            ellipsizeMode="middle"
            numberOfLines={1}
            style={
              Object {
                "color": "#4a4b4a",
                "fontSize": 18,
                "fontWeight": "600",
                "letterSpacing": -0.77,
              }
            }
          >
            Cadastro
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "alignSelf": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "flex-end",
            }
          }
        >
          <TouchableOpacity>
            <Text
              ellipsizeMode="clip"
              style={
                Object {
                  "color": "#ffffff",
                  "fontSize": 16,
                  "fontStyle": "normal",
                  "fontWeight": "600",
                  "letterSpacing": 0,
                }
              }
            />
          </TouchableOpacity>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 16,
      }
    }
  >
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flexDirection": "row",
            "height": 56,
            "paddingBottom": 16,
            "paddingLeft": 12,
            "paddingRight": 12,
            "paddingTop": 0,
          }
        }
      >
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
              },
              Object {
                "left": 8,
                "right": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "scaleX": 1,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "rgba(0, 0, 0, 0.38)",
                  "borderRadius": 4,
                  "borderStyle": "solid",
                  "borderWidth": 0.5,
                  "bottom": -4,
                  "left": -4,
                  "position": "absolute",
                  "right": -4,
                  "top": 0,
                }
              }
            />
          </View>
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": -4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "overflow": "hidden",
              "position": "absolute",
              "right": 0,
              "top": 4,
            }
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": 0.5,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": -4,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
                "width": 4,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": 0,
                "position": "absolute",
                "right": -4,
                "top": 0,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "alignSelf": "stretch",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "left": "-100%",
                "paddingLeft": "50%",
                "position": "absolute",
                "top": 0,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                  Object {
                    "translateY": 20,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
                "width": "200%",
              }
            }
          >
            <Text
              numberOfLines={1}
              onTextLayout={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.38)",
                  "fontSize": 16,
                  "includeFontPadding": false,
                  "lineHeight": 16,
                  "textAlign": "left",
                  "textAlignVertical": "top",
                }
              }
            >
              Nome
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              disableFullscreenUI={true}
              editable={true}
              keyboardType="default"
              onBlur={[Function]}
              onChange={[Function]}
              onChangeText={[Function]}
              onContentSizeChange={[Function]}
              onFocus={[Function]}
              rejectResponderTermination={true}
              selectionColor="rgb(0, 145, 234)"
              style={
                Array [
                  Object {
                    "flex": 1,
                    "includeFontPadding": false,
                    "margin": 0,
                    "padding": 0,
                    "paddingTop": 0,
                    "textAlign": "left",
                    "textAlignVertical": "top",
                    "top": 2,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, .87)",
                    "fontSize": 16,
                    "height": 24,
                  },
                  undefined,
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "flex-end",
              "minHeight": 8,
            },
            Object {
              "paddingLeft": 12,
              "paddingRight": 12,
            },
          ]
        }
      />
    </View>
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flexDirection": "row",
            "height": 56,
            "paddingBottom": 16,
            "paddingLeft": 12,
            "paddingRight": 12,
            "paddingTop": 0,
          }
        }
      >
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
              },
              Object {
                "left": 8,
                "right": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "scaleX": 1,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "rgba(0, 0, 0, 0.38)",
                  "borderRadius": 4,
                  "borderStyle": "solid",
                  "borderWidth": 0.5,
                  "bottom": -4,
                  "left": -4,
                  "position": "absolute",
                  "right": -4,
                  "top": 0,
                }
              }
            />
          </View>
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": -4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "overflow": "hidden",
              "position": "absolute",
              "right": 0,
              "top": 4,
            }
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": 0.5,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": -4,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
                "width": 4,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": 0,
                "position": "absolute",
                "right": -4,
                "top": 0,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "alignSelf": "stretch",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "left": "-100%",
                "paddingLeft": "50%",
                "position": "absolute",
                "top": 0,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                  Object {
                    "translateY": 20,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
                "width": "200%",
              }
            }
          >
            <Text
              numberOfLines={1}
              onTextLayout={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.38)",
                  "fontSize": 16,
                  "includeFontPadding": false,
                  "lineHeight": 16,
                  "textAlign": "left",
                  "textAlignVertical": "top",
                }
              }
            >
              E-mail
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="none"
              disableFullscreenUI={true}
              editable={true}
              keyboardType="email-address"
              onBlur={[Function]}
              onChange={[Function]}
              onChangeText={[Function]}
              onContentSizeChange={[Function]}
              onFocus={[Function]}
              rejectResponderTermination={true}
              selectionColor="rgb(0, 145, 234)"
              style={
                Array [
                  Object {
                    "flex": 1,
                    "includeFontPadding": false,
                    "margin": 0,
                    "padding": 0,
                    "paddingTop": 0,
                    "textAlign": "left",
                    "textAlignVertical": "top",
                    "top": 2,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, .87)",
                    "fontSize": 16,
                    "height": 24,
                  },
                  undefined,
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "flex-end",
              "minHeight": 8,
            },
            Object {
              "paddingLeft": 12,
              "paddingRight": 12,
            },
          ]
        }
      />
    </View>
    <View
      onResponderRelease={[Function]}
      onStartShouldSetResponder={[Function]}
      pointerEvents="auto"
    >
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flexDirection": "row",
            "height": 56,
            "paddingBottom": 16,
            "paddingLeft": 12,
            "paddingRight": 12,
            "paddingTop": 0,
          }
        }
      >
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
              },
              Object {
                "left": 8,
                "right": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "scaleX": 1,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "rgba(0, 0, 0, 0.38)",
                  "borderRadius": 4,
                  "borderStyle": "solid",
                  "borderWidth": 0.5,
                  "bottom": -4,
                  "left": -4,
                  "position": "absolute",
                  "right": -4,
                  "top": 0,
                }
              }
            />
          </View>
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "overflow": "hidden",
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": -4,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "overflow": "hidden",
              "position": "absolute",
              "right": 0,
              "top": 4,
            }
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": 0.5,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": -4,
              }
            }
          />
        </View>
        <View
          pointerEvents="none"
          style={
            Array [
              Object {
                "height": 4,
                "left": 0,
                "overflow": "hidden",
                "position": "absolute",
                "top": 0,
                "width": 4,
              },
              Object {
                "width": 8,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "borderColor": "rgba(0, 0, 0, 0.38)",
                "borderRadius": 4,
                "borderStyle": "solid",
                "borderWidth": 0.5,
                "bottom": -4,
                "left": 0,
                "position": "absolute",
                "right": -4,
                "top": 0,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "alignSelf": "stretch",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "left": "-100%",
                "paddingLeft": "50%",
                "position": "absolute",
                "top": 0,
                "transform": Array [
                  Object {
                    "scale": 1,
                  },
                  Object {
                    "translateY": 20,
                  },
                  Object {
                    "translateX": 0,
                  },
                ],
                "width": "200%",
              }
            }
          >
            <Text
              numberOfLines={1}
              onTextLayout={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.38)",
                  "fontSize": 16,
                  "includeFontPadding": false,
                  "lineHeight": 16,
                  "textAlign": "left",
                  "textAlignVertical": "top",
                }
              }
            >
              Telefone
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              autoCapitalize="sentences"
              disableFullscreenUI={true}
              editable={true}
              keyboardType="phone-pad"
              onBlur={[Function]}
              onChange={[Function]}
              onChangeText={[Function]}
              onContentSizeChange={[Function]}
              onFocus={[Function]}
              rejectResponderTermination={true}
              selectionColor="rgb(0, 145, 234)"
              style={
                Array [
                  Object {
                    "flex": 1,
                    "includeFontPadding": false,
                    "margin": 0,
                    "padding": 0,
                    "paddingTop": 0,
                    "textAlign": "left",
                    "textAlignVertical": "top",
                    "top": 2,
                  },
                  Object {
                    "color": "rgba(0, 0, 0, .87)",
                    "fontSize": 16,
                    "height": 24,
                  },
                  undefined,
                ]
              }
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "flex-end",
              "minHeight": 8,
            },
            Object {
              "paddingLeft": 12,
              "paddingRight": 12,
            },
          ]
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    style={
      Object {
        "padding": 16,
      }
    }
  >
    <TouchableOpacity
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#2f5199",
          "borderColor": "#dadada",
          "borderRadius": 40,
          "borderWidth": 0.3,
          "flexDirection": "row",
          "height": 52,
          "justifyContent": "center",
          "marginHorizontal": 0,
          "marginLeft": 0,
          "marginVertical": 5,
          "opacity": null,
          "paddingHorizontal": 24,
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ffffff",
              "fontSize": undefined,
              "paddingVertical": 10,
              "textAlign": "center",
            }
          }
        >
          Cadastrar
        </Text>
      </View>
    </TouchableOpacity>
  </View>
</View>
`;
